alfresco-actuators/src/main/resources/alfresco/templates/webscripts/eu/xenit/actuators/webscripts/health.get.json.ftl

```diff
@@ -0,0 +1,6 @@
+<#if healthInfo??>
+${healthInfo}
+<#else>
+missing
```

Comment:
Instead of printing a 'missing' string and still returning 200 OK, maybe we can fail instead? We are always expecting the health info to be set.

Reply:
Corrected.